models/stg_twilio.yml

```diff
@@ -6,8 +6,8 @@ models:
     tests:
       - dbt_utils.unique_combination_of_columns:
           combination_of_columns:
-            - account_id
-            - updated_at
+            - account_id 
+            - updated_at 
```

Comment:
I agree that we should test on the uniqueness of these two fields. However, should we also test that they are not null? I see that we test not_null for other models that only have one field.

Reply:
that is true, added!

Reply:
It looks like a not null was only added for the account_id, should you also add it for updated_at?

Reply:
added for updated_at